tests/util/DummyClam.php

```diff
@@ -71,7 +71,7 @@ protected function handleConnection($connection){
 				echo 'reading dataChunk' . PHP_EOL;
 				$dataChunk .= @fread($connection, $chunkSize);
 				echo 'dataChunk size '.strlen($dataChunk). PHP_EOL;
-			} while (is_resource($connection) && strlen($dataChunk) < $chunkSize);
+			} while (is_resource($connection) && strlen($dataChunk) !== $chunkSize);
```

Comment:
I should say that chunkSize is a packed binary that is sent before the chunk itself. so it is not that correct techically.

Reply:
this is what clamav expects as well

Reply:
@VicDeo 
tbh - I was just pocking your implementation to find the place where it fails - and at first it seemed like this because of:

![image](https://user-images.githubusercontent.com/6403243/40448155-fb033004-5ed4-11e8-9b10-b19ca2ab7d5f.png)

Unfortunately for whatever reason - I can't access the drone logs - and that loop fills up the log size of drone.

I've also seen that the chunksize is unpacked wrongly